### DISCUSSION TITLE
Lazyloading

### DISCUSSION THREAD

**Discussion by arthurhamon** [dimsemenov/PhotoSwipe#2036] [Q&A]
Hey,

Is their an option to do lazyloading on the image gallery?
My component will show the first image, then I want the second image to be loaded only when the gallery is open and user ask for second image.

Is this something already available or doable?

Thank you,
Arthur

**Comment by robots4life** [comment]
This might be what you are looking for [https://photoswipe.com/options/#preload](https://photoswipe.com/options/#preload).
